apps/service_providers/messaging_service.py

```diff
@@ -106,11 +113,11 @@ def client(self) -> TurnClient:
     def send_whatsapp_text_message(self, message: str, from_number: str, to_number):
         self.client.messages.send_text(to_number, message)
 
-    def send_whatsapp_voice_message(self, voice_audio: BytesIO, duration: int, from_number: str, to_number: str):
-        from apps.channels.audio import convert_audio
-
+    def send_whatsapp_voice_message(self, synthetic_voice: SynthesizedAudio, from_number: str, to_number: str):
         # OGG must use the opus codec: https://whatsapp.turn.io/docs/api/media#uploading-media
-        voice_audio = convert_audio(voice_audio, target_format="ogg", source_format="mp3", codec="libopus")
```

Comment:
Why are we converting to ogg here if the source format is already mp3?

Reply:
Just putting what we discussed here: TurnIO expects ogg (amongst other formats. I only got ogg working for now). We always convert to ogg to make sure it used the "libopus" codec.